httputil/retry.go

```diff
@@ -300,8 +301,8 @@ func IsCertExpiredOrNotValidYetError(err error) bool {
 	if urlErr, ok := err.(*url.Error); ok {
 		err = urlErr.Err
 	}
-	if certErr, ok := err.(x509.CertificateInvalidError); ok {
-		// Expired is misleading, also means not valid yet
```

Comment:
It looks like this comment got removed as part of the patch, would you mind keeping it (moving it close to the `certErr.Reason == x509.Expired` ideally)?

Reply:
Oops, probably typing too quick. Fixed.